florist/app/jobs/page.tsx

```diff
@@ -0,0 +1,135 @@
+"use client";
+import { ReactElement } from "react/React";
+import useJobStatus from "./hooks";
+
+const valid_statuses = {
+    NOT_STARTED: "Not Started",
+    IN_PROGRESS: "In Progress",
+    FINISHED_WITH_ERROR: "Finished with Error",
+    FINISHED_SUCCESSFULLY: "Finished Successfully",
+};
+
+interface JobData {
+    status: string;
+    model: string;
+    server_address: string;
+    server_info: string;
+    redis_host: string;
+    redis_port: string;
+    clients_info: Array<ClientInfo>;
+}
+
+interface ClientInfo {
+    client: string;
+    service_address: string;
+    data_path: string;
+    redis_host: string;
+    redis_port: string;
+}
+
+interface StatusProp {
+    status: string;
+}
+
+export default function Page(): ReactElement {
+    const status_components = Object.keys(valid_statuses).map((key, i) => (
```

Comment:
Code conventions for javascript are to have camel case instead of snake case on variable and function names, so `statusComponents` and `validStatuses` would be better names.

Reply:
There are a few other instances of this in this file and in the test file as well.

Reply:
Yeah I am half way between python and javascript land hahaha. Updated in most recent commit to use camel case throughout